### PR TITLE
Create release v0.0.36

### PR DESCRIPTION
**Pull request by sunker**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 All notable changes to this project will be documented in this file.
 
-## v1.0.0
+## v0.0.36
```
Comment (yaelleC) [inline CHANGELOG.md:5]: Is that to follow the same release # as `aws-sdk`?

**Review by yaelleC — APPROVED** [review]
LGTM, 
Just added a question around the release version number

File: CHANGELOG.md (hunk 1 above)
Comment (sunker) [reply CHANGELOG.md:5]: This _is_ actually the [grafana/aws-sdk](https://github.com/grafana/grafana-aws-sdk-react/packages/1188136). Now this repo contains the code for the npm package, but it used to be part of the https://github.com/grafana/grafana-aws-sdk repo until we decided to separate the frontend from the go code. 

The go package versioning does not sync with the npm package.

Comment (yaelleC) [reply CHANGELOG.md:5]: Aha, gotcha thanks!

**Pull request merged**